test/runtests.jl

```diff
@@ -44,15 +44,14 @@ T = Float32
         nt = NamedTuple(sim)
         @test sim == Simulation(nt)
         id = 2
-        deplV = estimate_depletion_voltage(sim, id, verbose = false)
-        @info deplV
-        @test isapprox(deplV, T(1904), atol = 2.0) 
+        deplV = SolidStateDetectors.estimate_depletion_voltage(sim, id, (verbose = false,))
```

Comment:
This function should be exported (but also doesn't harm to include this in the tests)